src/main/java/network/brightspots/rcv/DominionCvrReader.java

```diff
@@ -190,125 +190,156 @@ private void parseCvrFile(
     }
 
     try {
-      HashMap json = JsonParser.readFromFile(filePath, HashMap.class);
-      // top-level "Sessions" object contains a lists of Cvr objects from different tabulators
-      ArrayList sessions = (ArrayList) json.get("Sessions");
-      // for each Cvr object extract various fields
-      for (Object sessionObject : sessions) {
-        HashMap session = (HashMap) sessionObject;
-        // extract various ids
-        String tabulatorId = session.get("TabulatorId").toString();
-        String batchId = session.get("BatchId").toString();
-        Integer recordId = (Integer) session.get("RecordId");
-        String suppliedId = recordId.toString();
-        // filter out records which are not current and replace them with adjudicated ones
-        HashMap adjudicatedData = (HashMap) session.get("Original");
-        boolean isCurrent = (boolean) adjudicatedData.get("IsCurrent");
-        if (!isCurrent) {
-          if (session.containsKey("Modified")) {
-            adjudicatedData = (HashMap) session.get("Modified");
-          } else {
-            Logger.warning(
-                "CVR has no adjudicated rankings, skipping: "
-                    + "Tabulator ID: %s Batch ID: %s Record ID: %d",
-                tabulatorId, batchId, recordId);
-            continue;
+      Path singleCvrPath = Paths.get(manifestFolder, CVR_EXPORT);
+      Path firstCvrPath = Paths.get(manifestFolder, String.format(CVR_EXPORT_PATTERN, 1));
+      if (singleCvrPath.toFile().exists()) {
+        HashMap json = JsonParser.readFromFile(singleCvrPath.toString(), HashMap.class);
+        gatherCvrsForContest(json, castVoteRecords, contestIdToLoad, contestIdToCandidateCodes);
+      } else if (firstCvrPath.toFile().exists()) {
+        int recordsParsed = 0, recordsParsedAtLastlog = 0;
```

Comment:
There are a bunch of... pedantic coding standards from the VVSG which we need to follow.  So we are not allowed to initialize multiple variables in a single statement like this.  I think these can go away though if the cvr count logging logic is re-consolidated.

Reply:
Done